core/src/mast/serialization/decorator.rs

```diff
@@ -25,7 +25,7 @@ use crate::{
 #[derive(Debug, PartialEq, Eq)]
 #[cfg_attr(all(feature = "arbitrary", feature = "std"), derive(Arbitrary))]
 #[cfg_attr(
-    all(feature = "arbitrary", test),
+    all(feature = "arbitrary", feature = "std", test),
```

Comment:
Should this use `feature = "std"` or `feature = "serde"`? (and same for line 158 below)

I'm basign this purely on the test name - so, may be totally off here.

Reply:
`serde_test(false)`, for this macro, means this doesn't emit the serde tests.